compose.yaml

```diff
@@ -0,0 +1,19 @@
+services:
```

Comment:
By default the name of this file should `docker-compose.yaml` without what we must provide the name to the docker compose command

Reply:
From the compose spec compose.yaml is the preferred name ( https://github.com/compose-spec/compose-spec/blob/master/spec.md ) however for backward compatibility will rename it

Reply:
Interesting to know!